resources/validations/src/ssp.sch

```diff
@@ -355,7 +356,7 @@
                      value="./o:remarks =&gt; normalize-space()" />
             <sch:let name="remarks-length"
                      value="$remarks =&gt; string-length()" />
-            <sch:assert id="missing-response-components"
+            <sch:assert id="missing-response-components-1"
```

Comment:
It looks likes there is corresponding xspec that needs to be updated to reflect all the renames. https://github.com/18F/fedramp-automation/pull/104/checks?check_run_id=2850843050